Adding new sidebar page about extrusions to maps sdk section (#1205)
mapbox/android-docs: src/pages/maps/overview/data-driven-styling.md

/>
}}

### Fill extrusion

> Fill extrusion layers add 3D polygon "extrusions" to the map. This layer is often used to show 3D buildings, but any type of `Polygon` shape can be extruded with this layer type. (The Extrusions page)[/android/maps/overview/extrusions] has more information on styling 3D shape extrusions.

Reviewer: ```suggestion
Fill extrusion layers add 3D polygon "extrusions" to the map. This layer is often used to show 3D buildings, but any type of `Polygon` shape can be extruded with this layer type. [The Extrusions page](/android/maps/overview/extrusions) has more information on styling 3D shape extrusions.
```

Author: Fixed

> Fill extrusion layers add 3D polygon "extrusions" to the map. This layer is often used to show 3D buildings, but any type of `Polygon` shape can be extruded with this layer type. [The Extrusions page](/android/maps/overview/extrusions) has more information on styling 3D shape extrusions.

Reviewer: Hoping this will make tests pass: 

```suggestion
Fill extrusion layers add 3D polygon "extrusions" to the map. This layer is often used to show 3D buildings, but any type of `Polygon` shape can be extruded with this layer type. Find more information on styling 3D shape extrusions in the [Extrusions guide](/android/maps/overview/extrusions).
```

{{
<AndroidActivityToggle
  id="basic-extrusion-layer-setup"

java={`
mapboxMap.getStyle(new Style.OnStyleLoaded() {
	@Override
	public void onStyleLoaded(@NonNull Style style) {
		FillExtrusionLayer fillExtrusionLayer = new FillExtrusionLayer("extrusion-layer-id", "source-id");
		fillExtrusionLayer.setProperties(
		  fillExtrusionHeight(HEIGHT_NUMBER)
		);
		style.addLayer(fillExtrusionLayer);
	}
});
`}

kotlin={`
mapboxMap.getStyle {
	val fillExtrusionLayer = FillExtrusionLayer("extrusion-layer-id", "source-id")
	fillExtrusionLayer.setProperties(
		fillExtrusionHeight(HEIGHT_NUMBER)
	)
	it.addLayer(fillExtrusionLayer)
}
`}

> 

Author: Link added ✔️
